### DISCUSSION TITLE
Thank you devs for this awesome program!

### DISCUSSION THREAD

**Discussion by lua-cmd** [community/community#18446] [Other Feature Feedback, Questions, & Ideas]
Copilot is a great tool and I'm so happy to have it by my side, thank you devs.

Copilot hasn't failed me since and I even recommended it to all my friends. thanks a ton.
- lua-cmd

**Comment by Zerotask** [comment]
I also like it, being using it for 3 months now and I'm actively using it, e.g. do a line break and wait for the completion of Copilot
